### PR TITLE
Add descriptions

### PR DESCRIPTION
**Pull request by ja573**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
   - [582](https://github.com/thoth-pub/thoth/issues/582) - Add Crossmark metadata in Crossref DOI deposit when a Crossmark policy is present in the publisher record. Add Crossmark update new_edition metadata when a book is replaced by a new edition, and withdrawal metadata when a book is withdrawn from sale.
-  - [574](https://github.com/thoth-pub/thoth/issues/574) - Add descriptions to all remaining items in schema
+  - [574](https://github.com/thoth-pub/thoth/issues/574), [626](https://github.com/thoth-pub/thoth/pull/626) - Add descriptions to all remaining items in schema
 
 ### Fixed
   - [548](https://github.com/thoth-pub/thoth/issues/548) - Prevent users from deleting contributors/institutions which are linked to works by other publishers
+  - [626](https://github.com/thoth-pub/thoth/pull/626) - Rename `WorkStatus` `WITHDRAWN_FROM_SALE` to `WITHDRAWN`
```
Comment (ja573) [inline CHANGELOG.md:15]: You can just merge this with L:9 (by adding the link to this PR to the other line)

Comment (ja573) [reply CHANGELOG.md]: I meant, just add the PR number next to 595's (like in L:11) – there's no need to explain that you've renamed the work_status, you can just update the original message that mentions withdrawn from sale

**Comment by brendan-oconnell** [comment]
@ja573 I can't request you as a reviewer on your own PR, but I'd appreciate you reviewing my sections, especially my db migrations - thanks!

File: thoth-api/src/model/work/mod.rs [hunk 2]
```diff
@@ -75,7 +75,9 @@ pub enum WorkType {
 #[cfg_attr(
     feature = "backend",
     derive(DbEnum, juniper::GraphQLEnum),
-    graphql(description = "Publication status of a work"),
+    graphql(
+        description = "Publication status of a work throughout its lifecycle. For a visual representation of the workflow, refer to the work status flowchart https://github.com/thoth-pub/thoth/wiki/Thoth_Works#work-status-flowchart"
```
[the remark this answers is not in the document]
Comment (ja573) [reply thoth-api/src/model/work/mod.rs:79]: Which box is it? They display fine for me in firefox and safari

[the remark this answers is not in the document]
Comment (rhigman) [reply thoth-api/src/model/work/mod.rs:79]: `POSTPONED_INDEFINITELY`:

image

[the remark this answers is not in the document]
Comment (ja573) [reply thoth-api/src/model/work/mod.rs:79]: That's odd – does it display properly in the original https://github.com/thoth-pub/thoth/issues/595#issuecomment-2077651949 ? It should be exactly the same mermaid code

[the remark this answers is not in the document]
Comment (rhigman) [reply thoth-api/src/model/work/mod.rs:79]: Hah, yes, I'm getting the same display problem there, but I definitely didn't notice it when originally reading that discussion! Could be to do with updates in the meantime...

**Pull request merged**